dalek/cleaners/subscriptions_delete_net_app.go

```diff
@@ -26,128 +38,557 @@ func (p deleteNetAppSubscriptionCleaner) Name() string {
 }
 
 func (p deleteNetAppSubscriptionCleaner) Cleanup(ctx context.Context, subscriptionId commonids.SubscriptionId, client *clients.AzureClient, opts options.Options) error {
-	netAppAccountClient := client.ResourceManager.NetAppAccountClient
-	netAppCapcityPoolClient := client.ResourceManager.NetAppCapacityPoolClient
-	netAppVolumeClient := client.ResourceManager.NetAppVolumeClient
-	netAppVolumeReplicationClient := client.ResourceManager.NetAppVolumeReplicationClient
+	if accountLists, err := client.ResourceManager.NetAppAccountClient.AccountsListBySubscription(ctx, subscriptionId); err != nil {
+		return fmt.Errorf("listing NetApp Accounts for %s: %+v", subscriptionId, err)
```

Comment:
Please swap these to logs and change the logic around to a loop so it can just skip to `return nil`